tools/ci/build-openmc.sh

```diff
@@ -7,14 +7,17 @@ python tools/ci/gha-install.py
 
 # Create the caching folder
 mkdir openmc_src
-mkdir openmc_src/bin
 mkdir openmc_src/lib
+mkdir openmc_src/bin
 mkdir openmc_src/share
```

Comment:
Is there any reason the ordering of these two commands (making the lib and bin directories) matters?

Reply:
Thanks for making that consistent! I'll go ahead and merge this now.